lib/puppet/functions/extlib/has_module.rb

```diff
@@ -28,6 +30,7 @@ def has_module(module_name) # rubocop:disable Style/PredicateName
 
     metadata = JSON.parse(File.read(metadata_json))
     return true if metadata['name'] == full_module_name
+
     false
```

Comment:
This is even prettier:
```suggestion
    metadata['name'] == full_module_name
```

Reply:
https://github.com/voxpupuli/puppet-extlib/pull/178